bash_scripts/provision.sh

```diff
@@ -0,0 +1,318 @@
+#!/bin/bash
+set -e
+
+function init-variables {
+    local file="../env_variables"
+
+    CASSANDRA_REPLICATION_TYPE="Simple"
+    CASSANDRA_CONTACT_POINTS='172.16.238.68:9042' #initialized by the start-up script
+    CASSANDRA_REPLICAS="1"
+    POSTGRES_DRIVER_CLASS="org.postgresql.Driver"
+    POSTGRES_HOST='172.16.238.66' #initialized by the start-up script
+    POSTGRES_PWD="postgres"
+
```

Comment:
I think you should add:

    POSTGRES_PORT="5432"
    POSTGRES_USER="postgres"

as you refer to them on lines 117 and 118

Reply:
Yeah. This is read from the file. On lines 33 and 34